app.rb

```diff
@@ -0,0 +1,152 @@
+require_relative './options/add_music_album'
+require_relative './options/book_handler'
+require './classes/game'
+require './classes/author'
+require './modules/game_module'
+require './modules/author_module'
+require 'json'
+
+class App
+  include GamesDataController
+  include AuthorsDataController
+  def initialize
+    @book_handler = BookHandler.new
+    @games = load_games
+    @authors = load_authors
+  end
+
+  def display_menu
+    puts 'Select an option: '
+    puts
+    puts ['1 - List Books', '2 - List Music Albums', '3 - List Games', '4 - List Genres', '5 - List Labels',
+          '6 - List Authors', '7 - Add Book', '8 - Add Music Album', '9 - Add Game', '10 - Exit App']
+  end
```

Comment:
- [ ] When you click on the first 6 options, nothing is displayed when there are no data inserted already, so let us display a clear to the users when a user chose one of those menus before entering the data.

Reply:
Hello, @topeogunleye @vikipretium @Paskab012 I can see the message that is being displayed when a user chose to see or check the list of items when there is no data added.

![image](https://user-images.githubusercontent.com/55635977/171633128-6d0f20fe-f2a5-4cbc-ba8b-6dd11fc81654.png)

Reply:
Thanks, @Emmyn5600 👍🏼 We are grateful